src/Reports/SiteSummary.php

```diff
@@ -23,7 +23,21 @@ public function description()
         );
     }
 
-    public function sourceRecords($params)
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+
+        $this->beforeExtending('updateCMSFields', function (FieldList $fields) {
+            $fields->insertAfter('ReportDescription', new LiteralField(
+                'Version',
+                '<p><strong>' . _t(__CLASS__ . '.VERSION', 'Version: ') . $this->resolveCmsVersion() . '</strong></p>'
```

Comment:
I think we should make this an `<h3>` tag. There's an h2 in the report heading, so this would make sense as the next semantic step (and might align a little more closely with the designs)